pmusolino/Wormholy
# Fixed Swift 5.2 load issue, which doesn't fire anymore the initial load method

Fixes #78 

From Xcode 10.4 and Swift 5.2, there is an issue that doesn't allow Wormholy to work correctly since the initial method which uses an extension of UIApplication doesn't work anymore.

The new solution implemented is based on [this](https://stackoverflow.com/questions/49029515/is-there-an-alternative-to-initialize-in-macos-now-that-swift-has-deprecated-i/49078874#49078874).
This method works like a starter engine for Wormholy. We will call `appWillLaunch()` method in Swift (under the `StarterEngine` class), when `didFinishLaunchingWithOptions` is fired.
This is the flow:
1) App Start
2) `StarterEngine` receive a `UIApplicationDidFinishLaunchingNotification`
3) The notification fire the `appWillLaunch()` method under `SwiftySelfAwareHelper`

## Testing
1) Run the demo app from Xcode 10.4 (Swift 5.2)
2) Shake the device or the simulator
3) Make sure that Wormholy appears correctly now.

@kealdishx since you worked on the initial implementation https://github.com/pmusolino/Wormholy/pull/52 of the solution based on the UIApplication extension, I'd like it if you can check if everything continues to work correctly.
